frontend/src/metabase/redux/undo.unit.spec.ts

```diff
@@ -77,8 +79,9 @@ describe("metabase/redux/undo", () => {
 });
 
 const createMockStore = () => {
-  return configureStore({
+  const store = configureStore({
     // @ts-expect-error undo is still not converted to TS
     reducer: { undo: undoReducer },
   });
+  return store as typeof store & { dispatch: Dispatch };
```

Comment:
This should be the only test we have that tests the store directly like this, we put a band aid on it to not over-engineer something used once